src/components/visualization/bar-chart/BarChart.tsx

```diff
@@ -54,7 +52,7 @@ export default function BarChart({ chartInput }: { chartInput: ChartInput }) {
           <CartesianGrid strokeDasharray="3 3" />
           <XAxis dataKey={xAxis} label={xAxis} tick={false} />
           <YAxis type="number" domain={getDomain(chartInput.data, axesMap, xAxis)} ticks={getTicks()} />
-          <ReferenceLine y={0} stroke="#000" />
+          <ReferenceLine y={0} stroke="var(--bs-dark)" />
```

Comment:
Is this necessary? The rest of the chart's colors don't change with the theme, right? (Excluding the bars, of course)

Reply:
Agreed. Reverted